.github/workflows/action-conda-build.yml

```diff
@@ -0,0 +1,56 @@
+name: Conda-build
+
+# runs on a push on master and at the end of every day
+on:
+  # triggering on push without branch name will run tests everytime
+  # there is a push on any branch
+  # turn it on only if needed
+  push:
+    branches:
+    - master
+    - github-actions2
```

Comment:
perhaps add a note that this is a test branch

Reply:
good idea!